impeller/entity/shaders/texture_downsample.frag

```diff
@@ -7,11 +7,14 @@ precision mediump float;
 #include <impeller/constants.glsl>
 #include <impeller/types.glsl>
 
+layout(constant_id = 0) const float supports_decal = 1.0;
+
 uniform f16sampler2D texture_sampler;
 
 uniform FragInfo {
   float edge;
   float ratio;
+  float decal;
```

Comment:
```suggestion
  float use_decal;
```

Since we have to use float types as bool values, it helps to reinforce this isn't a continuous value like it seems.

Reply:
Done